vllm/v1/attention/backends/flash_attn.py

```diff
@@ -141,112 +140,50 @@ def forward(
         query = query.view(-1, self.num_heads, self.head_size)
         key = key.view(-1, self.num_kv_heads, self.head_size)
         value = value.view(-1, self.num_kv_heads, self.head_size)
-
-        output = torch.empty_like(query)
-        torch.ops.vllm.unified_v1_flash_attention(
-            output,
-            query,
-            key,
-            value,
-            self.num_heads,
-            self.head_size,
-            self.num_kv_heads,
-            kv_cache,
-            self.kv_cache_dtype,
+        output = output.view(-1, self.num_heads, self.head_size)
+
+        kv_cache_dtype: str = self.kv_cache_dtype
+        softmax_scale: float = self.scale
+        window_size: Optional[List[int]] = self.sliding_window
+        alibi_slopes: Optional[torch.Tensor] = self.alibi_slopes
+        logits_soft_cap: Optional[float] = self.logits_soft_cap
```

Comment:
What are these lines for?

Reply:
fixed in [c622271](https://github.com/vllm-project/vllm/pull/10822/commits/c6222710ecb84a3e0e91d50edcf2b2ddcccc9547)